braincloud/src/androidTest/java/com/bitheads/braincloud/services/TestFixtureBase.java

```diff
@@ -89,20 +95,20 @@ public boolean shouldAuthenticate() {
     private void LoadIds() {
         if (m_serverUrl.length() > 0) return;
 
-        File idsFile = new File("ids.txt");
+        Context ctx = InstrumentationRegistry.getInstrumentation().getContext();
+        InputStream is = null;
         try {
-            System.out.println("Looking for ids.txt file in " + idsFile.getCanonicalPath());
+            is = ctx.getAssets().open("ids.txt");
         } catch (IOException e) {
-            e.printStackTrace();
+            System.out.println("Error opening ids.txt");
+            throw new RuntimeException(e);
         }
 
-        if (idsFile.exists()) System.out.println("Found ids.txt file");
-
         List<String> lines = new ArrayList<>();
         BufferedReader reader = null;
 
         try {
-            reader = new BufferedReader(new FileReader(idsFile));
+            reader = new BufferedReader(new InputStreamReader(is));
```

Comment:
Minor: I'd convert this assignment into a "try with resources" assignment. This will ensure the stream gets closed at the end (not a real concern for a unit test that you know will be terminating but still...) 

i.e. 
```
try (BufferedReader reader = new BufferedReader(new InputStreamReader(is)) {
etc...
}
```